<ui version="4.0" >
 <class>TUview</class>
 <widget class="QWidget" name="TUview" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Ui_TUview</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>10</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible" >
       <bool>false</bool>
      </property>
      <widget class="QTextEdit" name="txtSource" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>221</red>
             <green>223</green>
             <blue>228</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Dark" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>85</red>
             <green>85</green>
             <blue>85</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>199</red>
             <green>199</green>
             <blue>199</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>225</red>
             <green>225</green>
             <blue>225</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Shadow" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>103</red>
             <green>141</green>
             <blue>178</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>82</red>
             <green>24</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>232</red>
             <green>232</green>
             <blue>232</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>221</red>
             <green>223</green>
             <blue>228</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Dark" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>85</red>
             <green>85</green>
             <blue>85</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>199</red>
             <green>199</green>
             <blue>199</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Shadow" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>103</red>
             <green>141</green>
             <blue>178</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>82</red>
             <green>24</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>232</red>
             <green>232</green>
             <blue>232</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>128</red>
             <green>128</green>
             <blue>128</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Button" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>221</red>
             <green>223</green>
             <blue>228</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Midlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Dark" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>85</red>
             <green>85</green>
             <blue>85</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Mid" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>199</red>
             <green>199</green>
             <blue>199</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>199</red>
             <green>199</green>
             <blue>199</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>128</red>
             <green>128</green>
             <blue>128</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Shadow" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Highlight" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>86</red>
             <green>117</green>
             <blue>148</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>0</red>
             <green>0</green>
             <blue>238</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>82</red>
             <green>24</green>
             <blue>139</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="AlternateBase" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>232</red>
             <green>232</green>
             <blue>232</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="tabStopWidth" >
        <number>79</number>
       </property>
      </widget>
      <widget class="QLabel" name="lblComment" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="txtTarget" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::DefaultContextMenu</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollBar" name="fileScrollBar" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>Navigate in your file</string>
     </property>
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="tracking" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtSource</tabstop>
  <tabstop>txtTarget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
